src/modules/auth/auth.service.ts

```diff
@@ -79,7 +79,9 @@ export class AuthService {
       token: this.jwtService.sign({
         sub: id,
         email,
+        role,
       }),
+      id,
```

Comment:
See if these are still necessary after the comments made in the [#32](https://github.com/ZenBit-Tech/HiveIn_fe/pull/32) PR